### PR TITLE
docs: added bigpanda.md event handler documentation

### PR DESCRIPTION
**Pull request by rhajek**
BigPanda event handler documentation.

_Describe your proposed changes here._

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/kapacitor/v1.5/event_handlers/bigpanda.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+| Value      | BigPanda Alert Field       |
+| ----       | ----        |
+| alert ID   | check |
+| message    | description |
+| details    | details |
+| alert task name  | task | 
+
+All EventData tags are also appended into BigPanda Alert as **Additional attributes**. See 
+[BigPanda Alert REST API](https://docs.bigpanda.io/reference#alerts) for more information.
```
Comment (sanderson) [inline content/kapacitor/v1.5/event_handlers/bigpanda.md:74]: - All EventData tags are ~also~ appended ~into~ **to the** BigPanda alert as **Additional attributes**

**Review by sanderson — CHANGES_REQUESTED** [review]
Thanks for this @rhajek. Just a few changes needed.

**Comment by docmerlin** [comment]
@sanderson go ahead and target master with this.  This isn't something we have to backport, so we use the easy workflow of treating master as tip.

**Comment by docmerlin** [comment]
On, My end, I don't see anything still glaring. Once Sanderson is happy, I'll go ahead and merge.

**Review by sanderson — APPROVED** [review]
Thanks @rhajek !

**Pull request merged**